wordle/__init__.py

```diff
@@ -93,6 +93,13 @@ def partial_match_exact_and_close():
     for word in ["agent", "burst", "canoe"]:
         check50.c.run(f"./wordle_test check_word arise {word}").stdout(3)
 
+
+@check50.check(compiles)
+def partial_match_repeat_guessed_letter():
```

Comment:
This is minor, but we tend to standardize function names to be consistent throughout the code.
`partial_match_repeat_guessed_letter` could just be `partial_match_repeat_letter` since it's implicit that the function is dealing with a guessed word.